### ISSUE TITLE
TrailingWhitespace rule should ignore indents in multiline string literals

### ISSUE THREAD

**Issue by elijah-pl** [arturbosch/detekt#2000]
The rule or changes you're looking for might have already been suggested!
Please search in the [issues](https://github.com/arturbosch/detekt/issues) before creating a new one.

## Expected Behavior of the rule
`TrailingWhitespace` rule should ignore indents in multiline string literals.
```kt
val example = """
    Should ignore indent on the next line
    
    Should ignore indent on the previous line
""".trimIndent()
```

## Context
Users should be able to format multiline string literals without breaking checkstyle rule.

**Comment by elijah-pl** [comment]
I can start my work on this issue as soon as maintainers confirm that it needs to be fixed.

**Comment by schalkms** [comment]
Thanks for reporting and willing to commit to detekt! 
We really appreciate any help from our community.
This issue is fixed by #1971 and included in the next release (1.1.1).
However, you could improve the multi-line test case in the `TrailingWhitespaceNegative` class.
